hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/store/contracts/precompile/HTSPrecompiledContract.java

```diff
@@ -227,6 +235,10 @@ public long gasRequirement(final Bytes bytes) {
     @Override
     public PrecompileContractResult computePrecompile(
             final Bytes input, @NonNull final MessageFrame frame) {
+        if (unqualifiedDelegateDetected(frame)) {
+            return INVALID_DELEGATE;
```

Comment:
The `setExceptionalHaltReason` call here has not been picked [from the other PR](https://github.com/hashgraph/hedera-services/pull/5567/files#diff-8700cc4c63009e10f6c47e97ed56bae8e59782b7550e20b3d664b21f5d184d46R238).

Reply:
Tyvm @dimitar-dinev, fixed!